doc/reference/networking/net_mgmt.rst

```diff
@@ -61,17 +63,6 @@ associated callback function is invoked with the actual event
 code. This makes it possible for different events to be handled by the
 same callback function, if desired.
 
-.. warning::
```

Comment:
Wouldn't this warning still be ok here?

Reply:
No, there is no layer/code false positive anymore since the change that made the callback unique on layer/code pair.

Reply:
Hmm, I thought we still need to check the exact value of the event like this `if (mgmt_event == NET_EVENT_IPV6_ADDR_ADD) {`, and this warning was about that?

Reply:
True, I misread it